okhttp-android-support/src/test/java/okhttp3/internal/huc/CacheAdapterTest.java

```diff
@@ -116,7 +116,7 @@
     };
     setInternalCache(new CacheAdapter(responseCache));
     client = client.newBuilder()
-        .sslSocketFactory(sslContext.getSocketFactory())
+        .sslSocketFactory(sslContextBuilder.socketFactory(), sslContextBuilder.trustManager())
```

Comment:
ok, so if you are suggesting moving this new Builder to okhttp/src/main then it also kills off DefaultSecurity or extra methods in Util.  Is that what you mean?